CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## lastest(master branch)
+
+## v0.5.0
+
+- Use MutationObserver to observe log for work instance_linker.js correctly [#68](https://github.com/cookpad/kuroko2/pull/68)
+- Support multiple hosted domains [#106](https://github.com/cookpad/kuroko2/pull/106)
+- Let all job instance be target of cleanup_old_instance script [#107](https://github.com/cookpad/kuroko2/pull/107), [#108](https://github.com/cookpad/kuroko2/pull/108), [#109](https://github.com/cookpad/kuroko2/pull/109)
+- Support script revision [#110](https://github.com/cookpad/kuroko2/pull/110)
+- Upgrade from factory_girl to factory_bot [#111](https://github.com/cookpad/kuroko2/pull/111)
```

Comment:
This change should not be included because it doesn't have any effect on kuroko2 users.

Reply:
4fe8b46 deleted